src/Input/ChoiceParam.php

```diff
@@ -33,12 +36,21 @@ public function getQuestion(): Question
     {
         $defaultValue  = $this->getDefault();
         $defaultPrompt = $defaultValue !== null
-            ? sprintf(' [<comment>%s</comment>]', $defaultValue)
+            ? sprintf(
+                ' [<comment>%s</comment>]',
+                is_array($defaultValue) ? implode(', ', $defaultValue) : $defaultValue
+            )
             : '';
+        $multiPrompt   = sprintf(
+            "\n(Multiple selections allowed; hit Return after each.%s Hit Return to stop prompting)\n",
+            $this->isRequired() ? ' At least one selection is required.' : ''
+        );
```

Comment:
One thing I noted when playing with the `ChoiceQuestion`: it does not present any information to the user whatsoever that they can (a) submit multiple values, and (b) _how_ they should submit them (the `ChoiceQuestion` normally wants them comma-separated when it is a `multiSelect`).

I've added verbiage here and in the `StandardQuestionTrait` around the ability to submit multiple values, and whether at least one is required.